irclib.py

```diff
@@ -363,53 +363,54 @@ class ServerNotConnectedError(ServerConnectionError):
     pass
 
 
-# Huh!?  Crrrrazy EFNet doesn't follow the RFC: their ircd seems to
-# use \n as message separator!  :P
+# Huh!? Crrrrazy EFNet doesn't follow the RFC: their ircd seems to
+# use \n as message separator! :P
 _linesep_regexp = re.compile("\r?\n")
 
 class ServerConnection(Connection):
     """This class represents an IRC server connection.
 
-    ServerConnection objects are instantiated by calling the server
-    method on an IRC object.
-    """
+ServerConnection objects are instantiated by calling the server
+method on an IRC object.
+"""
 
     def __init__(self, irclibobj):
         Connection.__init__(self, irclibobj)
-        self.connected = 0  # Not connected yet.
+        self.connected = 0 # Not connected yet.
         self.socket = None
-        self.ssl = None
+        self.sslsock = None
 
     def connect(self, server, port, nickname, password=None, username=None,
-                ircname=None, localaddress="", localport=0, ssl=False, ipv6=False):
+                ircname=None, localaddress="", localport=0, sslsock=False, ipv6=False):
         """Connect/reconnect to a server.
 
-        Arguments:
+Arguments:
 
-            server -- Server name.
+server -- Server name.
 
-            port -- Port number.
+port -- Port number.
 
-            nickname -- The nickname.
+nickname -- The nickname.
 
-            password -- Password (if any).
+password -- Password (if any).
 
-            username -- The username.
+username -- The username.
 
-            ircname -- The IRC name ("realname").
+ircname -- The IRC name ("realname").
 
-            localaddress -- Bind the connection to a specific local IP address.
+localaddress -- Bind the connection to a specific local IP address.
 
-            localport -- Bind the connection to a specific local port.
+localport -- Bind the connection to a specific local port.
 
-            ssl -- Enable support for ssl.
+sslsock -- Enable support for ssl.
```

Comment:
Now I'm wondering, did you just pull a new version of irclib.py? If so, that's fine, but please note that in the commit message and/or pull request description.